mmaction/datasets/pipelines/formating.py

```diff
@@ -220,8 +259,9 @@ class FormatShape:
         input_format (str): Define the final imgs format.
     """
 
-    def __init__(self, input_format):
+    def __init__(self, input_format, collapse=False):
```

Comment:
docstring for collapse

What does collapse mean?

Reply:
collapse means to collapse the NCTHW to CTHW if N == 1, which is the case when we train and test detectors. This change was made to be compatible with mmdet.